include/PikaRevision.h

```diff
@@ -0,0 +1,8 @@
+/* THIS FILE IS AUTOGENERATED - DO NOT EDIT */
```

Comment:
Just remove it from the commit and you can probably add it to the
.gitignore as well. We generate this file to keep track of what version of
MOOSE Pika is using.

Remove it by:
get reset -- include/PikaRevision.h

On Mon, Feb 9, 2015 at 8:14 AM, Micah Johnson <notifications@github.com>
wrote:

> In include/PikaRevision.h
> <https://github.com/idaholab/pika/pull/108#discussion_r24334243>:
>
> > @@ -0,0 +1,8 @@
> > +/* THIS FILE IS AUTOGENERATED - DO NOT EDIT */
>
> What am I supposed to do with this file? Delete it? I don't remember this
> ever showing up before...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/pika/pull/108/files#r24334243>.
>